### PR TITLE
backport: Add authorization header to houston notifications URL #794 …

### PR DESCRIPTION
**Pull request by andriisoldatenko**

### CODE REVIEW

File: charts/astronomer/values.yaml [hunk 1]
```diff
@@ -18,12 +18,12 @@ images:
     tag: 0.16.3
     pullPolicy: IfNotPresent
   registry:
-    repository: quay.io/astronomer/ap-registry
-    tag: 3.11.5-4
+    repository: astronomerinc/ap-registry
```
Comment (bote795) [inline charts/astronomer/values.yaml:21]: is this correct?

File: charts/astronomer/values.yaml [hunk 2]
```diff
@@ -18,12 +18,12 @@ images:
...
+    tag: 3.14.2-2
     pullPolicy: IfNotPresent
   houston:
-    repository: quay.io/astronomer/ap-houston-api
-    tag: 0.16.18
+    repository: asoldatenko/ap-houston-api
```
Comment (bote795) [inline charts/astronomer/values.yaml:25]: and here?

Comment (andriisoldatenko) [reply charts/astronomer/values.yaml:25]: temp fix to run it in CI, until houston PR in review https://github.com/astronomer/houston-api/pull/813

**Comment by andriisoldatenko** [comment]
@danielhoherd correct PR :^^^

File: charts/astronomer/values.yaml (hunk 1 above)
Comment (danielhoherd) [reply charts/astronomer/values.yaml:21]: Good catch. This is the old pre-quay registry. This should be reverted to quay.io.

**Review by danielhoherd — APPROVED** [review]
LGTM as long as we switch the docker registries back.

**Comment by bote795** [comment]
is it okay that the tests are failing?

**Comment by andriisoldatenko** [comment]
> is it okay that the tests are failing?

of course no, i'm working on fixing.

**Review by danielhoherd — APPROVED** [review]

**Comment by andriisoldatenko** [comment]
practically LGTM:
```
Every 2.0s: kubectl get pods -A                                                                                                                                                                                          Andriis-iPro: Wed Nov  3 00:57:10 2021

NAMESPACE            NAME                                                       READY   STATUS      RESTARTS   AGE
astronomer           astronomer-astro-ui-6d5dbdfb47-76t89                       1/1     Running     0          5m53s
astronomer           astronomer-cli-install-7bc5747466-77b4q                    1/1     Running     0          5m53s
astronomer           astronomer-commander-bc56cfdb4-dm4mh                       1/1     Running     0          5m53s
astronomer           astronomer-houston-b47644758-l5mdm                         1/1     Running     0          5m53s
astronomer           astronomer-houston-upgrade-deployments-22ddn               0/1     Completed   0          5m52s
astronomer           astronomer-kubed-7544557ddd-q72m7                          1/1     Running     0          5m53s
astronomer           astronomer-nginx-74f549ff6b-ph7mw                          1/1     Running     0          5m52s
astronomer           astronomer-nginx-default-backend-9d568cd6f-2t9mh           1/1     Running     0          5m53s
astronomer           astronomer-postgresql-0                                    1/1     Running     0          5m53s
astronomer           astronomer-prisma-599cc5b546-bgx4p                         1/1     Running     0          5m52s
astronomer           astronomer-prometheus-blackbox-exporter-57bd9484d7-mc75v   1/1     Running     0          5m52s
astronomer           astronomer-prometheus-blackbox-exporter-57bd9484d7-vcvx2   1/1     Running     0          5m52s
astronomer           astronomer-prometheus-node-exporter-89kr2                  1/1     Running     0          5m53s
astronomer           astronomer-registry-0                                      1/1     Running     0          5m53s
kube-system          coredns-66bff467f8-mwdzg                                   1/1     Running     0          5m53s
kube-system          coredns-66bff467f8-z492t                                   1/1     Running     0          5m53s
kube-system          etcd-kind-control-plane                                    1/1     Running     0          6m9s
kube-system          kindnet-gjlwc                                              1/1     Running     0          5m53s
kube-system          kube-apiserver-kind-control-plane                          1/1     Running     0          6m9s
kube-system          kube-controller-manager-kind-control-plane                 1/1     Running     0          6m9s
kube-system          kube-proxy-c8lkz                                           1/1     Running     0          5m53s
kube-system          kube-scheduler-kind-control-plane                          1/1     Running     0          6m8s
local-path-storage   local-path-provisioner-59c6df4d-5gnwv                      1/1     Running     0          5m53s
```

**Pull request merged**